src/dotnet/commands/RestoringCommand.cs

```diff
@@ -20,7 +20,8 @@ private IEnumerable<string> ArgsToForwardToRestore()
                 !a.StartsWith("/t:") &&
                 !a.StartsWith("/target:") &&
                 !a.StartsWith("/ConsoleLoggerParameters:") &&
-                !a.StartsWith("/clp:"));
+                !a.StartsWith("/clp:") &&
+                !a.StartsWith("/p:TargetFramework"));
```

Comment:
If this is supposed to catch arbitrary msbuild parameters then `/p:RuntimeIdentifier=osx.10.11-x64;TargetFramework=netcoreapp1.1` is probably going to break it.